runbooks/VMCannotBeEvicted.md

```diff
@@ -0,0 +1,22 @@
+# VMCannotBeEvicted
+
+## Meaning
+
+This alert fires when the eviction strategy of a VM is set to "LiveMigration" but the VM is not migratable.
+
+## Impact
+
+Such a none-migratable VMs block node eviction which can impact operations such as node drain, updates, etc..
```

Comment:
fixed

Reply:
@ezrasilvera I think this sentence would be clearer like this:
Non-migratable VMs block node eviction. This can impact operations such as node drain, updates, etc..

Reply:
@sradco Thanks. Will change.